src/commcare_cloud/fab/fabfile.py

```diff
@@ -168,11 +168,18 @@ def load_env():
 def _setup_env(env_name):
     env.env_name = env_name
     load_env()
+    _confirm_branch(env.default_branch)
     _confirm_environment_time(env_name)
     execute(env_common)
     execute(_confirm_deploying_same_code)
 
 
+def _confirm_branch(default_branch):
```

Comment:
nit: the name of this function says `confirm`, but it simply sets the attribute and prints a message. `_confirm_environment_time` prompts for a response (consistent with the conventional meaning of _confirm_).
```suggestion
def _set_code_branch(default_branch):
```